examples/notebooks/arvind_dendrite_role/generate_network2.sh

```diff
@@ -0,0 +1,29 @@
+export IPYTHONDIR="`pwd`/.ipython"
```

Comment:
is this file really necessary? It is basically a duplicate of the original file (I suspect this will clutter the directory)

Reply:
Removing